linker/Tests/Mono.Linker.Tests.Cases.Expectations/Metadata/KeepTypeForwarderOnlyAssembliesAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Mono.Linker.Tests.Cases.Expectations.Metadata {
+	[AttributeUsage (AttributeTargets.Class)]
+	public sealed class KeepTypeForwarderOnlyAssembliesAttribute : BaseMetadataAttribute {
```

Comment:
Agreed, the current pattern is tedious.  I'd rather not deal with it in this PR.  I have a rather large PR in progress to the UnityLinker and these are the upstream changes it depends on.  I'd rather not be forced into adding refactoring changes into my UnityLinker PR.

How about I improve the options in a separate PR?  That way I also have chance to sync up our linker with your recent BlackList change and this PR before doing refactoring.